lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -5626,32 +5626,31 @@ class DecomposeAtenAdaptiveAvgPool2dOp
                                  : rewriter.create<Torch::ConstantIntOp>(
                                        loc, rewriter.getI64IntegerAttr(
                                                 inputShape[rank - 2 + i])));
+        strides.push_back(constantOne);
```

Comment:
Now that strides == kernelSizeValues below, this seems to be the only place where those might differ, correct? Is it necessary to have stride = 1 in this case, or can you get rid of the separate stride prim list altogether and just pass the kernelSizeInts to the AtenAvgPool op?

Regardless, if you'd prefer to keep them separate for readability, I think that's fine. too.

Reply:
It is not necessary to discuss the case of unitOutputSize=1. So I eliminated the branch and modified the test case. What do you think about my latest change? By the way, thank you very much for your review.